src/functional/iterator/embedded.py

```diff
@@ -586,11 +606,28 @@ def shape(self):
         return self.array().shape
 
 
+def _is_tuple_axis(axis: Axis):
+    return axis is None
+
+
 def get_ordered_indices(
-    axes: Iterable[Axis], pos: Mapping[str, FieldIndex]
+    axes: Iterable[Axis], pos: Mapping[str, FieldIndex | SparsePositionEntry]
```

Comment:
Just a question: what does `pos: Mapping[str, FieldIndex | SparsePositionEntry]` represent here?

Reply:
changed to `Mapping[Tag, FieldIndex | SparsePositionEntry]`. `Tag` here is dimension tag.